terraform/gitlab/vpn/Makefile

```
@@ -12,7 +12,7 @@ server_cn = $(cn)-server-$(AZUL_DEPLOYMENT_STAGE)
 ca_cn = $(cn)-$(AZUL_DEPLOYMENT_STAGE)
 pki = $(azul_easyrsa_pki)/gitlab/$(AZUL_DEPLOYMENT_STAGE)
 
-export EASYRSA_CERT_EXPIRE = 365
+export EASYRSA_CERT_EXPIRE = 30 * 5
```

```suggestion
export EASYRSA_CERT_EXPIRE = 365 * 5 / 12
```

Are you sure that this is for both the server and the client certs?

---

Since `365 * 5 / 12` is an [invalid value](https://github.com/DataBiosphere/azul/pull/7238#issuecomment-3009205167) what would be an appropriate alternative?